traitsui/editor.py

```diff
@@ -598,3 +640,30 @@ def save_prefs(self):
         """ Returns any user preference information associated with the editor.
         """
         return None
+
+    # Utility context managers ----------------------------------------------
+
+    @contextmanager
+    def no_trait_update(self, name):
+        """ Context manager that blocks updates from the named trait. """
+        if self._no_trait_update is None:
```

Comment:
This branch looks unnecessary: I don't think `_no_trait_update` can ever be `None`.